pkgs/os-specific/linux/systemd/default.nix

```diff
@@ -137,7 +137,7 @@ stdenv.mkDerivation {
     owner = "systemd";
     repo = "systemd-stable";
     rev = "v${version}";
-    sha256 = "sha256-lfG6flT1k8LZBAdDK+cF9RjmJMkHMJquMjQK3MINFd8=";
+    sha256 = "sha256-000000000000000000cF9RjmJMkHMJquMjQK3MINFd8=";
```

Comment:
:eyes:

Reply:
Yes, still building :-D

Reply:
Silly! Why request me for review anyway?